starwars_api/models.py

```diff
@@ -28,8 +33,11 @@ def all(cls):
         later in charge of performing requests to SWAPI for each of the
         pages while looping.
         """
-        pass
-
+        if cls.RESOURCE_NAME == 'people':
+            return PeopleQuerySet()
```

Comment:
re: abstraction. completely agreed. which is why I hoped someone else would've figured out a better way.

@jangeador you're welcome, glad it helped. ^^ your feedback helps me practice my PR-reviewing and code-reading-and-understanding skills too.